apps/renterd/lib/multipartUpload.spec.ts

```diff
@@ -0,0 +1,311 @@
+import { Response, delay } from '@siafoundation/react-core'
+import { MultipartParams, MultipartUpload } from './multipartUpload'
+
+describe('MultipartUpload', () => {
+  it('should report progress and complete with serial chunks', async () => {
+    // note that the upload mock is configured to report progress 2 times per chunk
+    const params = getMockedParams({
+      file: new File(['01234567890123456789'], 'test.txt', {
+        type: 'text/plain',
+      }),
+      chunkSize: 2,
```

Comment:
For the production environment make sure that the size of a part equals `4MiB * minShards`. That way you avoid every part resulting in a partial slab.

Reply:
👍 go it, that value should be #500